Makefile

```diff
@@ -18,7 +18,7 @@ GO_LINKER_FLAGS_OPTIMIZATIONS = -s -w
 GO_LINKER_FLAGS = $(GO_LINKER_FLAGS_OPTIMIZATIONS) $(GO_LINKER_FlAGS_VARS)
 
 # variables that can be overridden by the user
-PREFIX ?= nginx-gateway-fabric## The name of the NGF image. For example, nginx-gateway-fabric
+PREFIX ?= ghcr.io/nginxinc/nginx-gateway-fabric## The name of the NGF image. For example, nginx-gateway-fabric
```

Comment:
This shouldn't be changed here. This refers to local images.

Reply:
This is my fault. I was helping Saloni and was trying to get the generated plus manifests to match the oss manifests and totally forgot about the private N+ registry.

Reply:
We have updated this